Support.md

```diff
@@ -2,6 +2,18 @@
 
 This project is open source, and is not a company. Instead we rely on a network of contributors and users to provide support.
 
+## Free support
+
+Problem ? Don't worry, someone else has probably been through that before you. 
+
+* First have a look at [NBitStack](https://nbitstack.com/c/btcpayserver) for a curated list of topics already answered by the community.
+
+* If your issue is not referenced there, or you want to feature request with BTCPay, please file a [github issue](https://github.com/btcpayserver/btcpayserver/issues).
```

Comment:
If your issue is not referenced there, **or you want to request a new feature** please **open** a [github issue](https://github.com/btcpayserver/btcpayserver/issues).